deliverables/README.md

```diff
@@ -20,49 +20,89 @@ I seek the Rails
+**What are forms?**<br>
+HTML forms are used to collect user input, and define an HTML form. <br>
+**What is a div?**<br>  
+divs are used in HTML and CSS documents. In HTML, they define a division or section in the document. In CSS, divs group block-elements in order to format them properly. <br>
+
+**What are selectors?**<br>
+Selectors are used in CSS and allow the developer to select and manipulate HTML elements. They can "find" HTML elements based on their ID's, Classes, or attributes. <br>
+**What are properties?**<br>
+Properties are special features that can be utilized to change the formatting of a specific element. These include colors, opacity, borders, lists, columns, etc. <br>
+**What are values?**<br>
+Values are assigned to the properties that give them a specific task. Ex: font-size 14px, or font-family Verdana.<br>
+**How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?**<br>
+There are a couple of ways for an element to be inherited; the default browsers markup settings, the styles can be specified by the user, or the styles can be linked to the document by the author, typically through an external style sheet, through a determiend style at the beginning of the document, or within a specific element in the body (inline). <br>
+**What are two CSS attributes you can change to push an element around on the page?**<br>
+Margin, fixed positioning, relative positioning, floating positioning. <br>
+**What are the three different ways to include CSS in your project or use CSS to style a particular element?**<br>
+In an external file, in a definition at the beginning of the document, or on a specific element in the body of the document. <br>
 
 ---
 ### Section 3: Git and Github  
-What is Git?  
-What is SCM?  
-What is a VCS?  
-Why is Git useful for a developer?  
-Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
-How do you create it on Github?  
-How do you commit changes?  
-What is the difference between staging and committing changes?  
-What is the difference between committing your changes and pushing them to Github?  
-What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
-What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
-What is "Cloning" a repo?  
+**What is Git?**<br>
+Git is a control system for software development. It holds a repository of projects.<br>
+
+**What is SCM?**<br>
+Software Configuration Management- it tracks and controls changes in software. <br>
+**What is a VCS?** <br>
+Veritas Cluster Server- it provides cluster capabilities to systems running other applications. <br>
```

Comment:
**V**erson **C**ontrol **S**ystem.  This includes things like Git, Google Docs (where you can see a history of changes) and Wikipedia (which also has a history of changes you can view)